RNAdam-core/src/main/scala/org/bdgenomics/RNAdam/algorithms/quantification/Quantify.scala

```diff
@@ -44,7 +44,7 @@ object Quantify extends Serializable with Logging {
             transcripts: RDD[Transcript],
             kmerLength: Int,
             maxIterations: Int,
-            calibrateKmerBias: Boolean = true): RDD[(Transcript, Double)] = {
+            calibrate: Boolean = true): RDD[(Transcript, Double)] = {
```

Comment:
I think it would be preferable to have seperate `calibrateKmerBias` and `calibrateLengthBias` flags, instead of a single `calibrate` flag.